src/codewhisperer/tracker/codewhispererCodeCoverageTracker.ts

```diff
@@ -224,26 +223,19 @@ export class CodeWhispererCodeCoverageTracker {
 
     public countTotalTokens(e: vscode.TextDocumentChangeEvent) {
         // ignore no contentChanges. ignore contentChanges from other plugins (formatters)
-        // only include contentChanges from user action
+        // only include contentChanges from user action.
+        // Also ignore deletion events due to a known issue of tracking deleted CodeWhiperer tokens.
```

Comment:
curious what's the known issue here

Reply:
seems we have some problem tracking deletion token so remove it for now?

Reply:
I have a CW suggestion accepted there -- upon deletion of some texts, I am able to track the # of characters got deleted, but not able to track the # of characters deleted dynamically. To actually track this needs another non-trivial work. Not part of this PR.